client/archonMaker.js

```diff
@@ -210,7 +210,7 @@ export const buildDeckList = async (deck, language, translate, allCards) => {
             y = y + 44;
         }
 
-        const rarity = new fabric.Image(
+        const rarity = await new fabric.Image(
```

Comment:
Are you sure this function is async, or returns a promise?

Reply:
You're right., it's a synchronous function.  Ill remove these